src/components/Layouts/MainLayout/MainLayout.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="main-layout">
+    <button-translation />
+    <aside class="main-sidebar">
+      <h1 class="main-sidebar__title">
+        <img
+          src="/img/appointment-logo.svg"
+          width="32"
+          height="30"
+          :title="$t('common.components.sidebar.titleImgAlt')"
+          :alt="$t('common.components.sidebar.titleImgTitle')"
+        />
+        <span class="main-sidebar__title-text">{{
+          $t('common.components.sidebar.title')
+        }}</span>
+      </h1>
+      <nav class="main-sidebar__main-nav">
+        <ul class="main-sidebar__nav-container">
+          <li>
+            <router-link :to="RoutePaths.APPOINTMENTS">
+              <as-nav-label
+                :label-text="
+                  $t('common.components.sidebar.navItems.appointments')
+                "
+                :active="
+                  router.currentRoute.value.name == RouteNames.HOME ||
+                  router.currentRoute.value.name == RouteNames.APPOINTMENTS
+                "
+                :right-icon-name="IconName.ANGLE_RIGHT"
+              />
+            </router-link>
+          </li>
+          <li>
+            <router-link :to="RoutePaths.USERS">
+              <as-nav-label
```

Comment:
if all as-nav-label have a router link, why dont put the router link in to the component?

Reply:
Done!!! Thanks!!